pkg/services/playlist/playlistimpl/sqlx_store.go

```diff
@@ -39,12 +39,12 @@ func (s *sqlxStore) Insert(ctx context.Context, cmd *playlist.CreatePlaylistComm
 
 		if len(cmd.Items) > 0 {
 			playlistItems := make([]playlist.PlaylistItem, 0)
-			for _, item := range cmd.Items {
+			for order, item := range cmd.Items {
 				playlistItems = append(playlistItems, playlist.PlaylistItem{
 					PlaylistId: p.Id,
 					Type:       item.Type,
 					Value:      item.Value,
-					Order:      item.Order,
+					Order:      order + 1,
```

Comment:
bug fix?  maybe -- the "Update" flavor uses the index order, but "Insert" will use the raw order value.

Regardless for sure a very very low priority issues!